Content.Shared/Construction/Conditions/EntityWhitelistCondition.cs

```diff
@@ -11,6 +11,8 @@ namespace Content.Shared.Construction.Conditions;
 [DataDefinition]
 public sealed partial class EntityWhitelistCondition : IConstructionCondition
 {
+    [Dependency] private readonly EntityWhitelistSystem _whitelistSystem = default!;
```

Comment:
a

Reply:
this one's a little bit more tricky since it doesn't have an entity manager already in the args...